financial-services-accelerator/components/org.wso2.financial.services.accelerator.gateway/src/test/resources/test-validation-response.json

```diff
@@ -0,0 +1,12 @@
+{
+    "isValid": true,
+    "errorCode": "",
+    "errorMessage": "",
+    "httpCode": "",
+    "modifiedPayload": {
+        "key": "value"
+    },
+    "consentInformation": {
+        "key": "value"
+    }
+}
```

Comment:
Missed EOF

Reply:
Fixed in https://github.com/wso2/financial-services-accelerator/pull/130/commits/1e13ec5af065388311c13d8ca7f04ef804d84184